boards/arm/efr32_thunderboard/efr32bg22_brd4184b.dts

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
This file looks identical to `boards/arm/efr32_thunderboard/efr32bg22_brd4184a.dts`. Please separate their common parts into a separate dtsi, which is later included in the board dts.

Reply:
@fkokosinski ok, done (common parts in `efr32bg22_brd4184.dtsi`).

Reply:
Thanks!

Can you modify the dtsi hierarchy so that it's linear? e.g. `efr32bg22_brd4184b.dts` -> `efr32bg22_brd4184.dtsi` -> `thunderboard.dtsi`?

Reply:
@pepe2k yes